docs/spark/tutorials/get-started.md

```diff
@@ -134,7 +140,7 @@ The `dotnet` command creates a `new` application of type `console` for you. The
 
 To use .NET for Apache Spark in an app, install the Microsoft.Spark package. In your command prompt, run the following command:
 
-`dotnet add package Microsoft.Spark --version 0.6.0`
+`dotnet add package Microsoft.Spark --version 0.8.0`
 
 ### 3. Code your app
```

Comment:
Can we change the following code part to explicit variable type?
`var spark = SparkSession...` => `SparkSession spark = SparkSession...`
`var words = dataFrame...` => `DataFrame words = dataFrame...`